enterprise_access/apps/api/v1/views/content_assignments/assignments_admin.py

```diff
@@ -124,12 +124,12 @@ def get_queryset(self):
             pass
 
         # Annotate extra dynamic fields used by this viewset for DRF-supported ordering and filtering,
-        # but only for the list and retrieve actions:
+        # but only for the list, retrieve, and cancel/remind-all actions:
         # * learner_state
         # * learner_state_sort_order
         # * recent_action
         # * recent_action_time
-        if self.action in ('list', 'retrieve'):
+        if self.action in ('list', 'retrieve', 'remind_all', 'cancel_all'):
```

Comment:
For cancel/remind-all, we need to filter on the dynamic fields that admins can filter on via the admin portal, so include those in the base queryset for the cancel/remind-all actions.